io-package.json

```diff
@@ -266,7 +267,7 @@
         "password"
     ],
     "native": {
-        "ip": "fritz.box",
```

Comment:
This change is breaking! Why it is needed?

Reply:
To set the correct ip/host in locallink (io-package.json) it is necessary to rename it, because %ip% will be replaced with the ip from the iobroker host.

Reply:
Ok, but then as written in my last comments we need to make sure it is done in a backward compatible way and this means "migration of the data in that variable". Please

Reply:
Could you give me an example of how do you imagine that?

Reply:
In fact in index_m and main.js check for ala "if (iporhost === undefined && ip !== undefined) iporhost = ip"